runtime-local/test/integration/integration_test.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+from pathlib import Path
+from re import compile, Pattern
+from subprocess import PIPE, Popen
+from threading import Timer
+
+
+def create_dummy_vspec_file():
+    """Creates an empty vspec json file in the project's cache directory
+    plus the respective 'vspec_file_path' cache variable pointing to it.
+    !!! Todo: Replace this workaround solution making assumptions about
+              the cache location.
+    """
+    cache_paths = Path("~/.velocitas/projects").expanduser().rglob("cache.json")
+    for cache_path in cache_paths:
+        vspec_path = cache_path.parent / "dummy_vspec.json"
+        with open(vspec_path, "w") as dummy_vspec_file:
+            dummy_vspec_file.write("{}\n")
```

Comment:
Instead of creating a file on the fly you could place a prepared file in the test directory, next to .velocitas.json.

But for use cases like this I could envision there is need for a `cache` subcommand in CLI, e.g. `velocitas cache --get-path` or `velocitas cache --get-content` to dump either the full path to a project's cache or the entire cache content to stdout

Reply:
Or even the option to get and set cache variables, e.g. `velocitas cache get {var_name}` / `velocitas cache set {var_name} {value}` (?)

Reply:
Yes, we've been planning that, too!